apps/server/src/modules/tool/controller/mapper/external-tool-request.mapper.spec.ts

```diff
@@ -15,23 +14,20 @@ import {
 	SortOrderMap,
 	ToolConfigType,
 } from '@shared/domain';
-import {
-	ExternalToolSortOrder,
-	SortExternalToolParams,
-} from '@src/modules/tool/controller/dto/request/external-tool-sort.params';
-import { ExternalToolSearchParams } from '@src/modules/tool/controller/dto/request/external-tool-search.params';
-import { CustomParameterTypeParams } from '../interface/custom-parameter-type.enum';
-import { CustomParameterCreateParams } from '../controller/dto/request/custom-parameter.params';
-import { BasicToolConfigParams } from '../controller/dto/request/basic-tool-config.params';
-import { Oauth2ToolConfigParams } from '../controller/dto/request/oauth2-tool-config.params';
-import { ExternalToolParams } from '../controller/dto/request/external-tool-create.params';
+import { ExternalToolSortOrder, SortExternalToolParams } from '../dto/request/external-tool-sort.params';
+import { ExternalToolSearchParams } from '../dto/request/external-tool-search.params';
+import { CustomParameterTypeParams } from '../../interface/custom-parameter-type.enum';
+import { CustomParameterCreateParams } from '../dto/request/custom-parameter.params';
+import { BasicToolConfigParams } from '../dto/request/basic-tool-config.params';
+import { Oauth2ToolConfigParams } from '../dto/request/oauth2-tool-config.params';
+import { ExternalToolParams } from '../dto/request/external-tool-create.params';
 import { ExternalToolRequestMapper } from './external-tool-request.mapper';
-import { Lti11ToolConfigParams } from '../controller/dto/request/lti11-tool-config.params';
-import { TokenEndpointAuthMethod } from '../interface/token-endpoint-auth-method.enum';
-import { CustomParameterLocationParams } from '../interface/custom-parameter-location.enum';
-import { CustomParameterScopeParams } from '../interface/custom-parameter-scope.enum';
-import { LtiMessageType } from '../interface/lti-message-type.enum';
-import { LtiPrivacyPermission } from '../interface/lti-privacy-permission.enum';
+import { Lti11ToolConfigParams } from '../dto/request/lti11-tool-config.params';
+import { TokenEndpointAuthMethod } from '../../interface/token-endpoint-auth-method.enum';
+import { CustomParameterLocationParams } from '../../interface/custom-parameter-location.enum';
+import { CustomParameterScopeParams } from '../../interface/custom-parameter-scope.enum';
+import { LtiMessageType } from '../../interface/lti-message-type.enum';
+import { LtiPrivacyPermission } from '../../interface/lti-privacy-permission.enum';
```

Comment:
Can we use barrel files here too (like in `N21-410-ctl-delete`)?

Reply:
It does not make sense to do the same change twice. I will make sure it is right when merging `main` into `N21-410-ctl-delete`

Reply:
Ok, agreed.